### PR TITLE
Enable automation for YaST in Container in ALP in OSD

### PR DESCRIPTION
**Pull request by tinawang123**
Enable automation for YaST in Container in ALP in OSD

- Related ticket: https://progress.opensuse.org/issues/130384
- Verification run: https://openqa.suse.de/tests/overview?distri=alp&version=1.0&build=4.1&groupid=496
- MR: https://gitlab.suse.de/qsf-y/qa-sle-functional-y/-/merge_requests/509

### CODE REVIEW

**Review by lemon-suse — APPROVED** [review]

File: schedule/alp/alp_yast_as_workload.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: alp_yast_as_workload
+description: >
+    Enable YaST automation test for ALP
+vars:
+    YUI_REST_API: 1
+schedule:
+    - installation/bootloader_uefi
+    - microos/selfinstall
```
Comment (jknphy) [inline schedule/alp/alp_yast_as_workload.yaml:9]: for Flavor: Micro-Default seems that we need a different schedule file

File: schedule/alp/alp_yast_as_workload.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+name: alp_yast_as_workload
+description: >
+    Enable YaST automation test for ALP
```
Comment (jknphy) [inline schedule/alp/alp_yast_as_workload.yaml:4]: doesn't reflect that we "run YaST as a workload for ALP"

**Comment by jknphy** [comment]
- Could you clean the job group removing the one with previous name https://openqa.suse.de/tests/11342389 ?

File: schedule/alp/alp_yast_as_workload.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
```
Comment (jknphy) [inline schedule/alp/alp_yast_as_workload.yaml:1]: it is already inside /alp/ folder, should be just `yast_as_workload.yaml`

File: schedule/alp/alp_yast_as_workload.yaml (hunk 1 above)
Comment (tinawang123) [reply schedule/alp/alp_yast_as_workload.yaml:9]: for Flavor: Micro-Default, I used 'schedule/alp/alp_yast.yaml' schedule file.
Please check the setting: https://openqa.suse.de/tests/11345346#settings

File: schedule/alp/alp_yast_as_workload.yaml (hunk 2 above)
Comment (tinawang123) [reply schedule/alp/alp_yast_as_workload.yaml]: Done

File: schedule/alp/alp_yast_as_workload.yaml (hunk 3 above)
Comment (tinawang123) [reply schedule/alp/alp_yast_as_workload.yaml]: Done

**Pull request merged**